scripts/__init__.py

```diff
@@ -1,3 +1,6 @@
+import os
+import sys
```

Comment:
> scripts/__init__.py

AFAIK, it is not a standalone script. It is used through eval()

Reply:
[eval() is evil](https://nedbatchelder.com/blog/201206/eval_really_is_dangerous.html) ;-) but if these imports have already been done then these lines are no-ops but if the imports have not been done then these lines will prevent crashes.

Reply:
@cclauss, you mean it prevents crashes in a linting job?

Reply:
Crashes in any situation where `os` and `sys` have not already been imported by some other file.

As it says in The Zen of Python", _explicit is better than implicit._